libs/custom-paper-handler/src/printing.ts

```diff
@@ -150,10 +149,6 @@ export function imageDataToBinaryBitmap(
 export function chunkBinaryBitmap(
   binaryBitmap: BinaryBitmap
 ): PaperHandlerBitmapExt[] {
-  if (binaryBitmap.width > DEVICE_MAX_WIDTH_DOTS) {
-    throw new Error("binaryBitmap width exceeds printer's allowed size");
-  }
```

Comment:
This is already checked [in driver.printChunk()](https://github.com/votingworks/vxsuite/blob/f09310f2cf6bb1d1f4b43650cf335ee86930839c/libs/custom-paper-handler/src/driver/driver.ts#L810-L813).